lib/class/video.class.php

```diff
@@ -255,7 +255,7 @@ public function format($details = true)
         $this->f_title      = scrub_out($this->title);
         $this->f_full_title = $this->f_title;
         $this->link         = AmpConfig::get('web_path') . "/video.php?action=show_video&video_id=" . $this->id;
-        $this->f_link       = "<a href=\"" . $this->link . "\" title=\"" . $this->f_title . "\"> " . $this->f_title . "</a>";
+        $this->f_link       = "<a href=\"" . $this->link . "\" title=\"" . scrub_out($this->f_title) . "\"> " . scrub_out($this->f_title) . "</a>";
```

Comment:
Is this at all necessary?

In line 255, we do: `$this->f_title = scrub_out($this->title);`

And in line 258 we then do: `$this->f_link = ... scrub_out($this->f_title)` ... essentially it means that we are doing:

`scrub_out(scrub_out($this->title))`

Also, I am not sure if it s indeed fixed in core because in my set-up, I would see the names of clips showing with the re-escaped characters.